src/app/courses/courses-list/courses-list.component.html

```diff
@@ -1 +1,8 @@
-<p>courses-list works!</p>
+<ul class="coursesList">
+    <li *ngFor="let item of courses; trackBy: trackByFn">
+        <app-course-item [id]="item.Id" [title]="item.Title" [description]="item.Description" [duration]="item.Duration"
+            [creationDate]="item.CreationDate" (deletedItemId)="onDeleteCourse($event)">
```

Comment:
For better readability if there are many inputs you can write each input from the new line
[id]="item.id"
[description]="item.description"

Reply:
I did it, but my plugins return the initial formatting after saving (